stories/index.stories.tsx

```diff
@@ -84,6 +70,35 @@ visualize('storybook-snapper', () => {
   story('Basic', () => {
     snap('sync example', () => <ExampleComponent />);
     snap('async example', done => <ExampleComponent onDone={done} />);
+
+    snap(
+      'run before',
+      <div style={{ fontFamily: 'Helvetica, Arial, sans-serif' }}>
+        <h3>Story is Waiting</h3>
+        <div className={styles.box}>
+          <img src={THUMBS_DOWN_SRC} alt="Thumbs" />
+        </div>
+        <p>Story will appear to not be ready without the <pre style={{display: 'inline'}}>runBefore</pre> script</p>
+      </div>,
+      async ({ rootEl }) => {
+        const img = rootEl.querySelector('img') as HTMLImageElement;
```

Comment:
any reason not to write a `react` code here?

Reply:
This is code that eyes-storybook runs in the browser.
So i'm not sure it will be possible.
Also there's no way to debug this: https://github.com/applitools/eyes-storybook/issues/30